.github/workflows/testing-and-deployment.yml

```diff
@@ -166,7 +166,7 @@ jobs:
             numpy-version: "default"
           - python-version: "3.12"
             vtk-version: "latest"
-            numpy-version: "pre"
+            numpy-version: "nightly"
```

Comment:
Renamed this for clarity. We can name it back when we revert to pre-releases once 2.0 is released (if we want to do that).